napari/viewer.py

```diff
@@ -161,3 +161,15 @@ def close(self):
     def __str__(self):
         """Simple string representation"""
         return f'napari.Viewer: {self.title}'
+
+
+def _convert_app_for_tracing(app):
+    """Return a new QApplicationWithTracing if we don't have it already.
+    """
+    from ._qt.qt_event_tracing import convert_app_for_tracing
+
+    # For tracing Qt events we need a special QApplication. If using
+    # `gui_qt` we already have the special one, and no conversion is done
+    # here. However when running inside IPython or Jupyter this is where we
+    # switch out the QApplication.
+    return convert_app_for_tracing(app)
```

Comment:
Why did you pull this out into a pass-through function...? The deleted code above seems clearer than this, which adds another layer of indirection...

Reply:
I agree the one line `_convert_app_for_tracing` is not needed. I think I had it doing more at first and it whittled down to just the one line. I'll get rid of it.

Reply:
I removed it.